app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: callable) -> callable:
+    results = {}
+
+    def wrapper(*args, **kwargs) -> callable:
+        key = args + tuple(sorted(kwargs.items()))
+        if key in results:
+            print("Getting from cache")
+            return results[key]
+        else:
```

Comment:
Delete this `else` statement, code after the `return` statement will not execute anyway.